doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -41,29 +41,45 @@ nRF52840 DK RAM and flash memory requirements
-| Router     | :ref:`CoAP Client <coap_client_sample>` | Yes             |                       458 |                           144 |                        32 |                        83 |                            23 |              634 |              106 |
-+------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+------------------+------------------+
-| End Device | :ref:`CoAP Client <coap_client_sample>` | No              |                       254 |                             0 |                        32 |                        80 |                             0 |              286 |               80 |
-+------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+------------------+------------------+
-| End Device | :ref:`CoAP Client <coap_client_sample>` | Yes             |                       399 |                           145 |                        32 |                        80 |                            23 |              576 |              103 |
-+------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+------------------+------------------+
++------------+-----------------------------------------+-------------------+-----------------+-----------------------------------+-------------------------------+---------------------------+-----------------------------------+-------------------------------+----------------------+------------------+----------------------+------------------+
+| Role       | Sample                                  | Library variant   | Multiprotocol   |   ROM OpenThread stack + App [kB] |   ROM Bluetooth LE stack [kB] |   Persistent storage [kB] |   RAM OpenThread stack + App [kB] |   RAM Bluetooth LE stack [kB] |   Avaialble ROM [kB] |   Total ROM [kB] |   Available RAM [kB] |   Total RAM [kB] |
++============+=========================================+===================+=================+===================================+===============================+===========================+===================================+===============================+======================+==================+======================+==================+
+| Router     | :ref:`CLI <ot_cli_sample>`              | Master            | No              |                               448 |                             0 |                        32 |                                92 |                             0 |                  480 |             1024 |                   92 |              256 |
```

Comment:
I think, that Available RAM [kB] is in fact total RAM used at least it looks like it contains the sum of Thread stack ram usage and BT ram

Reply:
Same goes for Available ROM

Reply:
yes, fixed